src/main/java/ui/windows/StatsController.java

```diff
@@ -1,54 +1,63 @@
 package ui.windows;
 
+import use_cases.DataAccess;
+import use_cases.PlayerInformation;
+
+import java.io.IOException;
+
 /**
  * Handles in- and outgoing data for a StatsDisplay.
  */
 public class StatsController {
     private StatsDisplay display;
+    private DataAccess database;
 
     /**
      * Construct a StatsController with no endpoint. 
      */
-    public StatsController() { 
+    public StatsController(DataAccess database) {
         this.display = null;
+        this.database = database;
     }
 
     /**
      * Sets a display endpoint for the controller.
-     * @param display The display to be used.
+     * @param display the display to be used
      */
     public void setDisplay(StatsDisplay display) {
         this.display = display;
     }
 
     /**
      * Requests statistics for a given user and updates display.
-     * @param username The name of the user.
-     * @return false if no display is supplied, true otherwise.
+     * @param username the name of the user
+     * @return false if no display is supplied, true otherwise
      */
-    public boolean tryRequestUser(String username) {
+    public boolean tryRequestUser(String username) throws IOException {
```

Comment:
What does the exception here do? The Javadoc should contain "@throws" if the method throws an exception.